books/RayTracingInOneWeekend.html

```diff
@@ -1073,29 +1073,29 @@
 the chapter.)
 
 <div class='together'>
-Pick a random point $s$ from the unit radius sphere that is tangent to the hitpoint, and send a ray
-from the hitpoint $\mathbf{p}$ to the random point $\mathbf{S}$. That sphere has center
-$\mathbf{p} + \vec{\mathbf{N}}$:
+Pick a random point s from the unit radius sphere that is tangent to the hitpoint, and send a ray
+from the hitpoint $p$ to the random point $s$. That sphere has center $(p + N)$:
 
-  ![Figure [rand-diffuse]: Generating a random diffuse bounce ray](../images/fig.rand-diffuse.png)
+  ![Figure [rand-vector]: Generating a random diffuse bounce ray](../images/fig.rand-vector.jpg)
 
 </div>
 
 <div class='together'>
-We also need a way to pick a random point in a unit radius sphere centered at the origin. One way to
-do that is to pick a random angle $a$ on the $\mathbf{Z}$ plane and a random $z$ coordinate in
-$[-1,+1]$ to construct a vector with some trigonometry:
+We also need a way to pick a random point in a unit radius sphere centered at the origin. We’ll use
+what is usually the easiest algorithm: a rejection method. First, we pick a random point in the unit
+cube where x, y, and z all range from -1 to +1. We reject this point and try again if the point is
```

Comment:
First, ~we~ pick a random point ...
~We~ Reject this point and try again if ...